src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -251,5 +258,216 @@ public void GetAsync_CredentialIsCredentialCacheUriRedirect_HttpStatusCodeOK()
                 await AssertSuccessfulGetResponse(response, HttpTestServers.SecureRemoteGetServer, _output);
             }
         }
+
+        [Fact]
+        public async Task PostAsync_CallMethod_StringContent()
+        {
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    var data = "Test String";
+                    var stringContent = new StringContent(data, UnicodeEncoding.UTF8, mediaTypeJson);
+                    HttpResponseMessage response =
+                        await client.PostAsync(HttpTestServers.RemotePostServer, stringContent);
+                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    var responseContent = response.Content.ReadAsStringAsync().Result;
```

Comment:
If the test method is marked as 'async', then yes, I agree that `await` should be used instead of .Result.